excel_import_export_sample/import_export_sale_order/actions.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Sales Order Import/Export -->
+    # TODO: should we use our own place to keep data instead of ir.attachment
```

Comment:
Use proper xml comment format

Reply:
I would add this remark in the README file section ROADMAP

Reply:
OK. I have removed those unnecessary comment, as they were already implemented.